### PR TITLE
Update to Qt Creator section of tutorial

### PR DESCRIPTION
**Pull request by dakcarto**
WIP, up to the `Building QGIS` subsection. Continues work started by @timlinux in #12.

Unintentionally conflicts with changes proposed in #13 (simultaneous drafts).

### CODE REVIEW

File: development/README.md [hunk 1]
```diff
@@ -286,167 +286,309 @@ $BUILD_SCRIPTS/qgis-dev-install.sh $BUILD_DIR
...
-**Note:** Most of these settings were taken from $(brew --repository qgis/qgisdev)/scripts/qgis-cmake-setup.sh. You can comment out the last line of the aforementioned script and then just echo the CMD instead of running it to get the equivalent options above.
+* **CMake Tool:** Brew CMake (which you previously created)
+* **CMake Generator:** CodeBlocks - Unix Makefiles
+* **CMake Configuration:** Press the `Change...` button and paste this into the box provided:
 
+  ```
+  CMAKE_CXX_COMPILER:STRING=%{Compiler:Executable}
+  CMAKE_C_COMPILER:STRING=%{Compiler:Executable:C}
+  QT_QMAKE_EXECUTABLE:STRING=%{Qt:qmakeExecutable}
+  CMAKE_BUILD_TYPE:STRING=RelWithDebInfo
```
Comment (timlinux) [inline development/README.md:406]: Won't this get set when you set up your project?

Comment (dakcarto) [reply development/README.md:406]: Yes. I think I added this prior to realizing that the Configure Project panel offered the selection of build configurations _prior_ to configuring the project (I think it used to just run default config first).

Comment (dakcarto) [reply development/README.md:406]: Also, I think it did not even work, since the project configuration always overrides it.

**Review by timlinux — APPROVED** [review]
LGTM - I will resolve any conflicts with my PR or cancel it as needed once yours is applied.

**Comment by timlinux** [comment]
@dakcarto I hope its is OK - I am going to merge this as people are still getting derailed by the old instructions I think....

**Pull request merged**